vsintegration/src/FSharp.Editor/CodeFix/AddOpenCodeFixProvider.fs

```diff
@@ -131,15 +131,16 @@ type internal FSharpAddOpenCodeFixProvider
                 openNamespaceFix context ctx name ns multipleNames)
             |> Seq.toList
             
-        let quilifySymbolFixes =
+        let qualifiedSymbolFixes =
             candidates
+            |> Seq.filter (fun (entity,_) -> not(entity.FullRelativeName.StartsWith("op_"))) // Don't include qualified operator names. The resultant codefix won't compile because it won't be an infix operator anymore.
```

Comment:
👍, thanks.  I can revert it back to `Contains` - only concern there is if something has "op_" in its name and *isn't* an operator.  Is that valid?  I wouldn't think it is.  Could also find the position of the last `.` and then check `StartsWith` on the substring that comes after.